main.css

```diff
@@ -10,4 +10,7 @@
     height: 100px;
     width: 100px;
     float: left;
-}
```

Comment:
This looks good. Ome small point is that it isn't formatted. Try and install a css formatted from the vs code extensions. There is no functional difference it just makes it easier to read.

Reply:
This isn't on my version.. Why is that? I know you talked about this before but I don't know why its on the PR but not on my version

Reply:
Check your commits are in sync with the remote. Do git log --oneline on your machine  then check the commits on this PR. If they're not the same then  do a push